main/file.c

```diff
@@ -43,22 +44,18 @@ static struct{
     uint8_t tx_buffer[MAX_TX_BUFFER_SIZE];
     size_t tx_data_size;
     esp_now_send_status_t last_message_status;
-    uint16_t interval_ms;
+    sub_status_t status;
```

Comment:
Możesz zmienić na state, bo status może być czymś innym, a jak mówimy o state to wiadomo, że chodzi o stan rakiety.

Reply:
A może i nawet lepiej current_state; A i nie powinien być typu sub_transmit_period_t?